ansible/roles/netplan/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Copyright (C) 2015-2017 Maciej Delmanowski <drybjed@gmail.com>
+# Copyright (C) 2015-2017 Robin Schneider <ypid@riseup.net>
+# Copyright (C) 2015-2017 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-only
+
+- name: Check netplan config and apply
+  ansible.builtin.command: /usr/sbin/netplan generate
+  notify: [ 'Apply netplan config' ]
+
+- name: Apply netplan config
+  ansible.builtin.command: /usr/sbin/netplan apply
```

Comment:
The handlers could be moved to the `global_handlers` role, that way other roles could use them in the future without requiring the `netplan` role as a dependency. Make sure that your role imports the `global_handlers` role for this to work.

Reply:
done